pallets/doas/src/lib.rs

```diff
@@ -115,7 +112,6 @@ pub mod pallet {
           let dispatch_info = call.get_dispatch_info();
           (
             dispatch_info.weight
-              .saturating_add(10_000)
```

Comment:
same here, can you remember why this changed?

Reply:
I should have checked but I was getting these errors and wasn't sure how to sort it
`sp_weights::weight_v2::Weight
pub const fn saturating_add(self, rhs: Self) -> Self

Saturating [Weight](vscode-file://vscode-app/Users/MAddis/Documents/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) addition. Computes self + rhs, saturating at the numeric bounds of all fields instead of overflowing.
Go to [Weight](vscode-file://vscode-app/Users/MAddis/Documents/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)
error: mismatched types
label: arguments to this function are incorrect
note: associated function defined here
label: arguments to this function are incorrect`

Reply:
Back in now